cli/inagoctl.go

```diff
@@ -59,6 +70,7 @@ func init() {
 	MainCmd.PersistentFlags().StringVar(&globalFlags.FleetEndpoint, "fleet-endpoint", "unix:///var/run/fleet.sock", "endpoint used to connect to fleet")
 	MainCmd.PersistentFlags().BoolVar(&globalFlags.NoBlock, "no-block", false, "block on synchronous actions or not")
 	MainCmd.PersistentFlags().BoolVarP(&globalFlags.Verbose, "verbose", "v", false, "verbose output or not")
+	MainCmd.PersistentFlags().BoolVar(&globalFlags.LogColor, "log-color", false, "whether to use colorful output for logs")
```

Comment:
I dislike the extra flag here. Can't we detect if the output is a tty device and if so enable color, otherwise disable it.
https://github.com/fatih/color/tree/master/vendor/github.com/mattn/go-isatty